python/GangaLHCb/test/Unit/DiracAPI/TestDiracCommands.py

```diff
@@ -55,6 +55,7 @@ def load_config():
 
 @pytest.yield_fixture(scope='class')
 def dirac_job(load_config):
+    from GangaDirac.Lib.Credentials.DiracProxy import DiracProxy
```

Comment:
Is there a good reason to have this import here? If so fine but in general it violates PEP8